Packages/Tests/SnapshotsTests/ConnectWalletSnapshotsTests.swift

```diff
@@ -78,7 +78,7 @@ final class ConnectWalletSnapshotsTests: XCTestCase {
     func test_connectingWallet_dark() async {
         let connectWalletView = ConnectingWalletView(selectedWallet: .metamask, cancelPressed: {})
             .environment(\.colorScheme, .dark)
-        XCTExpectFailure("Fails in github actions environment. Should pass locally")
+
         assertSnapshot(
             of: connectWalletView,
             as: .image(perceptualPrecision: 0.98, layout: .device(config: .iPhoneSe)))
```

Comment:
you should likely to change also `config`. However, you can consider using smaller images (stick with ipnone SE) to have smaller git repo size.